packages/amplication-client/src/App.tsx

```diff
@@ -40,6 +47,21 @@ function App() {
     setKeepLoadingAnimation(false);
   }, []);
 
+  const [, setInvitationToken] = useLocalStorage(
```

Comment:
dont use state to set invitation token. Add it to local storage and refresh page. It is a bad practice and any way you need to refresh it to trigger all API calls